setup.py

```diff
@@ -119,14 +119,14 @@ def find_version(*file_paths):
         "setuptools==59.5.0"
     ],
     "onnx": [
-        "torch==1.9.1",
-        "torchvision==0.10.1",
-        "onnx==1.11.0",
+        "torch==1.13.0",
```

Comment:
We need torch 1.12 to be aligned with the PT backend.

Reply:
Why should we align with PT backend here?

Reply:
after offline discussion, we decided to downgrade torch to 1.12.1

Reply:
Done